### PR TITLE
Tree-sitter running fixes for September

### PR DESCRIPTION
**Pull request by savetheclocktower**
Hopefully fewer in this one than in last month's.

### CODE REVIEW

**Comment by savetheclocktower** [comment]
I never put this into draft status, but if I had, I'd be taking it out right now. Let's get this reviewed so we can land it before the next release.

File: packages/language-shellscript/grammars/modern-tree-sitter-bash.cson [hunk 1]
```diff
@@ -21,6 +21,8 @@ firstLineRegex: [
 injectionRegex: '(^(bash|BASH)$|sh^|SH^)'
 
 treeSitter:
+  # Grammar generated on 2023-09-22 from commit:
```
Comment (confused-Techie) [inline packages/language-shellscript/grammars/modern-tree-sitter-bash.cson:24]: Do you want to modify this to instead use `parserSource` as we are now attempting to standardize it? (Also if you want we can look at getting #740 merged soon to properly enforce these new standards?

**Review by confused-Techie — APPROVED** [review]
LGTM!

As I'm missing context on some of these changes, I'll review what I know.

The changes here look very valid, and seem to correlate properly with actually issues being reported.

I've added a single comment for something we have discussed previously, so would be nice to address that.

And something not at all needed here, but I do want to point out, the usage of `var` in `tree-sitter.js` does somewhat hurt to look at, but I know that's pretty out of scope to fix here.

Otherwise, I want to confirm, @savetheclocktower can I link the following issues to this PR so they get closed on merge:

* #236 
* #736

**Comment by savetheclocktower** [comment]
One of those two issues is already linked, and the other one has already been closed.

**Review by confused-Techie — APPROVED** [review]
@savetheclocktower Thanks for adding the `parserSource` here.
And you're totally right, my bad no clarification needed on the linked issues.

Lets get this one merged!

**Pull request merged**